### DISCUSSION TITLE
[Feature Request] Visual feedback when saving media

### DISCUSSION THREAD

**Discussion by NMBlanco** [rodit/SnapMod#116] [Ideas & Feature Suggestions]
With SnapTools, there was an option to enable a visual feedback whenever you save a snap/story etc.. 
Like the screen would gently flash white/phone would give vibrate feedback when the saving was successfull.
It's very minor I know but I think it was kind of cool

Also something else that's minor, an option to automatically save every sent snap and save to an extra directory.

**Comment by RevealedSoulEven** [comment]
That's not required at all

**Comment by rodit** [comment]
Although not required, I think it would be useful for some feedback (even just a toast) which can be turned on to show that something has been downloaded as it's hard to tell whether it has worked sometimes. I will probably add that at some point.

**Comment by Magna-Graecia** [comment]
I personally already get notifications in the the form of the blue "save" bar that shows when using "save to camera roll", and a little download arrow that shows in the status bar for a brief moment when saving snaps/stories using the in-snap/in-story context menu, but I understand that everyone has a different device, so I can see this being useful. If you're going to do this, just make it a small little signal toast that displays for a brief second in the corner. SnapFreedom had a colored-coded dot toast that displayed for a second in the bottom-left of the screen. Green means it saved correctly. I think Orange meant that it saved but it was a duplicate due to already having been saved previously. And Red means it didn't save due to some issue. Just keep in mind that not everyone has a device with vibrate, and I personally wouldn't use it anyway because I find vibrate to be an unnecessary use of battery power for a use like this. A visual feedback in the moment after you pressed "save" is plenty lol.

**Comment by rodit** [comment]
Added an optional toast (see the 'Downloads' section of SnapMod's settings) to show when downloads start (v1.6.3)
